docs/guides/universal-profile/transfer-lyx.md

```diff
@@ -116,7 +116,7 @@ We can use the [`encodeABI()`](https://web3js.readthedocs.io/en/v1.7.4/web3-eth-
 
 ```javascript
 import Web3 from 'web3';
-const web3 = new Web3('https://rpc.l14.lukso.network');
+const web3 = new Web3('https://rpc.l16.lukso.network');
```

Comment:
I think this example works with profiles deployed on L14, this is why we kept l14 here. @fhildeb will probably know better.

Reply:
All good, the transfer lyx guide is already using the L16 faucet and should be updated. seems like this is a leftover that was missed.